src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVMDiagnosticsExtension.md

```diff
@@ -25,7 +25,7 @@ You must pass the output of this cmdlet to the Update-AzureRmVM cmdlet to implem
 
 ### Example 1: Remove the Diagnostics extension from a virtual machine
 ```
-PS C:\>Remove-AzureRmVMDiagnosticsExtension -ResourceGroupName "ResourceGroup11" -VMName "ContosoVM22" | Update-AzureRmVM
```

Comment:
The remove command itself is enough. Update-AzureVM is needed for V1 vm, but not V2 vm. Running the current sample command will give such error:
Update-AzureRmVM : The input object cannot be bound to any parameters for the command either because the command does not take pipeline input or the input and its properties do not match any of the 
parameters that take pipeline input.